pyproject.toml

```diff
@@ -26,7 +26,8 @@ classifiers = [
 dependencies = [
   "numpy",
   "torch",
-  "cvxopt",
+  "cvxpy-base",
```

Comment:
Why not cvxpy?

Reply:
CVXPY comes with multiple solvers by default. `cvxpy-base` is only the modeling language itself, and we only install one solver (`clarabel`) explicitly. This makes maintenance a lot easier.